### ISSUE TITLE
Import declaration adding multiple commas, formatting issue

### ISSUE THREAD

**Issue by mattcraigfleming** [mattpocock/boilersuit#4]
![screenshot 2018-09-07 at 10 43 05](https://user-images.githubusercontent.com/18464148/45211678-d5fa9400-b28a-11e8-931b-4f1aa057a612.png)

**Comment by mattcraigfleming** [comment]
excuse the laziness, thats a hard one to explain :)

**Comment by mattpocock** [comment]
Note to self - this appears to be an issue with adding properties to a list of already destructured imports

**Comment by mattpocock** [comment]
Potential fix: before adding new stuff, rewrite whole file to only use vertical-style imports, such as:
```javascript
import {
  something,
  somethingElse,
} from 'here';
```

**Comment by mattpocock** [comment]
This is still an issue when there is a default selector, for instance:

```javascript
import defaultSelector from './selectors';
```

This gets mangled.

It also has an error when it has to import a new set of destructured properties.

**Comment by mattpocock** [comment]
Fixed as of version 0.1.4.
